components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -513,26 +546,37 @@ func (backend *ESClient) JobStatus(ctx context.Context, jobID string) (project_u
 	}, nil
 }
 
-func getPercentageComplete(status interface{}) float64 {
+func getPercentageComplete(status interface{}) (float64, bool) {
```

Comment:
This function is used for other types of async jobs. Therefore we need to add the create and deletes to the completed amounts.

Reply:
There is also the case that the "created", "deleted", and "updated" are missing from the status. In that case, we return a false to signal that the percentage should not be used, but it is not an error.